src/store/actions.js

```diff
@@ -1,46 +1,88 @@
-import Xo from 'xo-lib'
-import { createBackoff } from 'jsonrpc-websocket-client'
-
-// ===================================================================
-
-const createAction = (() => {
-  const { defineProperty } = Object
-  const identity = payload => payload
-
-  return (type, payloadCreator = identity) => defineProperty(
-    (...args) => ({
-      type,
-      payload: payloadCreator(...args)
-    }),
-    'toString',
-    { value: () => type }
-  )
-})()
```

Comment:
Why not keep this function?

It permits creating both the action type and its creator at the same type which avoid mistakes.